wd_demo.au3

```diff
@@ -1251,8 +1258,11 @@ EndFunc   ;==>SetupChrome
 
 Func SetupEdge($bHeadless)
 	_WD_Option('Driver', 'msedgedriver.exe')
-	_WD_Option('Port', 9515)
-	_WD_Option('DriverParams', '--verbose --log-path="' & @ScriptDir & '\msedge.log"')
+	Local $iPort = _WD_GetFreePort(9515)
```

Comment:
Use a range here instead of a single port

Reply:
take a look: [$iPort uasge](https://github.com/Danp2/au3WebDriver/pull/519/commits/44384b84d308fac3a31e53fb0077b05bc9c6cb8b)